<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-8OV7IEGI" ignored="3" name="Regression Test" tests="5" failures="1" timestamp="2020-05-31T14:40:33 EDT" time="64.601" errors="0">
  <testcase name="addEmployee" time="13.635" classname="com.hrms.testcases.AddEmployeeTest"/>
  <testcase name="addEmployee" time="12.445" classname="com.hrms.testcases.AddEmployeeTest"/>
  <testcase name="addEmployee" time="8.937" classname="com.hrms.testcases.AddEmployeeTest"/>
  <testcase name="emptyUsernameLogin" time="0.353" classname="com.hrms.testcases.LoginTest"/>
  <testcase name="invalidPasswordLogin" time="0.91" classname="com.hrms.testcases.LoginTest">
    <failure type="java.lang.AssertionError" message="Error message text is not matched expected [Invalid credential] but found [Invalid credentials]">
      <![CDATA[java.lang.AssertionError: Error message text is not matched expected [Invalid credential] but found [Invalid credentials]
at com.hrms.testcases.LoginTest.invalidPasswordLogin(LoginTest.java:37)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- invalidPasswordLogin -->
  <testcase name="@AfterTest writeReport" time="0.0" classname="com.hrms.testbase.BaseClass">
    <ignored/>
  </testcase> <!-- @AfterTest writeReport -->
  <testcase name="validAdminLogin" time="0.0" classname="com.hrms.testcases.LoginTest">
    <ignored/>
  </testcase> <!-- validAdminLogin -->
  <testcase name="@BeforeTest generateReort" time="0.0" classname="com.hrms.testbase.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeTest generateReort -->
</testsuite> <!-- Regression Test -->
